api/Console.json

```diff
@@ -210,10 +216,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Console/countReset",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "68"
```

Comment:
I've confirmed this with http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8416 and BrowserStack.